alcs-frontend/src/app/services/application/decision/application-decision-v2/application-decision-v2.service.ts

```diff
@@ -156,4 +156,14 @@ export class ApplicationDecisionV2Service {
   async cleanDecisions() {
     this.$decisions.next([]);
   }
+
+  async getNextAvailableNumberInResolutionYear(resolutionYear: number) {
```

Comment:
I would just call this getNextAvailableReolutionNumber(), the fact that it takes year as an input should be a good enough indicator of how it works.